v2/cli/init_command_test.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNormalizeLanguage(t *testing.T) {
+	testCases := []struct {
+		name     string
+		expected string
+	}{
+		{"typescript", "ts"},
+		{"ts", "ts"},
+		{"javascript", "js"},
+		{"js", "js"},
```

Comment:
I think I noticed this in other places, too. This is a good habit to break. Be explicit about what field receive what values instead of implicitly relying on field order. If someone swapped lines 15 and 16 above, your tests would break.

If your test cases looked like this, that swap wouldn't hurt you:

```
{name: "typescript", expected: "ts"},
{name: "ts", expected: "ts"},
{name: "javascript", expected: "js"},
{name: "js", expected: "js"},
```

Not to mention this is more readable / less ambiguous for someone who comes along later.